docs/en/04_Performance_Guide/02_HTTP_Caching.md

```diff
@@ -59,30 +59,31 @@ To help explain how actively the content could be cached on CWP let's split the
 
 ### Configuration via headers
 
-Your best approach in controlling the caching behaviour is setting your response headers in accordance with the [RFC-2616](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html). 
+Your best approach in controlling the caching behaviour is setting the `Cache-Control` response header (see [Google's Web Fundaments - HTTP Caching](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching)) 
 This will ensure that all caches on the way of the response will be able to make reasonable decisions. This includes CWP's Local Cache, the CDN (Incapsula), any other public proxies (such as corporate gateways) and the browser cache.
 
 We will now explain some simple techniques on how to increase your cache utilisation. Let's have a look at the mapping between the cache levels and specific response headers.
 
 | Cache level | _Cache-Control_ header | _Vary_ header | Appropriate for |
 | - | - | - |
-| None | "no-cache, max-age=0" | _n/a_ | all |
-| Light | "max-age=X" | "Cookie, […]" | all |
-| Full | "max-age=X" | _none_ or "Accept-Encoding" | non-varying* |
+| None | "no-store, no-cache, must-revalidate" | "X-Forwarded-Protocol, X-Requested-With" | all |
+| Light | "max-age=X, must-revalidate" | "X-Forwarded-Protocol, X-Requested-With, […]" | all |
+| Full | "max-age=X" | _none_ | non-varying* |
 
-The easiest way improve caching on your dynamic responses is to use the [Controller Policy module](https://github.com/silverstripe/silverstripe-controllerpolicy). It allows you to customise the response headers per `Controller` without the need to modify any PHP code.
-For deeper customisations, you also [set HTTP Cache headers directly](https://docs.silverstripe.org/en/4/developer_guides/performance/http_cache_headers/).
+In CWP 2.1+ developers can make use of the `HTTPCacheControlMiddleware` to control the `Cache-Control` header added to
+responses. This is covered in more detail in the SilverStripe docs on []HTTP cache headers](https://docs.silverstripe.org/en/4/developer_guides/performance/http_cache_headers/#cache-control-headers).
 
 #### Defaults
 
 With the basic recipe all SilverStripe Framework responses come with the following _Cache-Control_ directive. You will see this response headers coming from the default recipe site.
 This means the response is classified as the "None" cache level.
 
 ```
-Cache-Control: no-cache, max-age=0, must-revalidate, no-transform
+Cache-Control: no-store, no-cache, must-revalidate
```

Comment:
double check this is the default

Reply:
checked - but this needs to change `no-store` is a really bad default...

Reply:
This is still the default for `STATE_ENABLED` (which is the default state). This isn't the place to discuss changing that, so I'm ignoring this as part of the peer review.